binder/src/test/java/io/grpc/binder/internal/PendingAuthListenerTest.java

```diff
@@ -26,12 +43,15 @@
 @RunWith(JUnit4.class)
 public final class PendingAuthListenerTest {
 
+  private static final MethodDescriptor<Void, Void> TEST_METHOD =
+      TestMethodDescriptors.voidMethod();
+
   @Rule public final MockitoRule mocks = MockitoJUnit.rule();
+  @Rule public final GrpcCleanupRule grpcCleanupRule = new GrpcCleanupRule();
 
   @Mock ServerCallHandler<Object, Object> next;
   @Mock ServerCall<Object, Object> call;
   @Mock ServerCall.Listener<Object> delegate;
-  @Captor ArgumentCaptor<Status> statusCaptor;
 
   private final Metadata headers = new Metadata();
   private final PendingAuthListener<Object, Object> listener = new PendingAuthListener<>();
```

Comment:
I'm not sure if we can stop mocking `ServerCall` for the remaining untouched tests; I don't know how else we could enforce that the current methods are called in that order, specially the cancellation.